### PR TITLE
Finger - Add scaling settings

### PR DESCRIPTION
**Pull request by Drofseh**
**When merged this pull request will:**
- add two settings to scale the size of the visual marker; `sizeCoef` will scale the size directly from the setting value, default 1x is the same as the current size; `proximityScaling` will scale the size based on the distance between the observer and the pointer, 2x the size at `maxRange`, 0.25x at 0m, interpolated in between.

### CODE REVIEW

**Comment by Salluci** [comment]
Nice workaround with `_sourceUnit`.

File: addons/finger/functions/fnc_perFrameEH.sqf [hunk 1]
```diff
@@ -34,6 +34,11 @@ private _iconSize = BASE_SIZE * 0.10713 * (call EFUNC(common,getZoom));
         // Fade out:
         _drawColor set [3, ((_drawColor select 3) * ((_timeLeftToShow min 0.5) / 0.5))];
 
+        private _iconSize = _iconBaseSize;
+        if (GVAR(proximityScaling)) then {
```
Comment (PabstMirror) [inline addons/finger/functions/fnc_perFrameEH.sqf:38]: I think we could get away with just a single setting?
```suggestion
        if (GVAR(sizeCoef) != 1) then {
```

Comment (Drofseh) [reply addons/finger/functions/fnc_perFrameEH.sqf:38]: I want people to be able to set a custom scale if they want, and also to enable/disable distance based scaling

Comment (PabstMirror) [reply addons/finger/functions/fnc_perFrameEH.sqf:38]: nm I guess they do different things

**Comment by Drofseh** [comment]
[Video of prox scaling](https://i.gyazo.com/415bfda72be4a95be108155501fd93db.mp4)

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**